plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultJdbcQueryBuilder.java

```diff
@@ -144,7 +144,7 @@ public void setup()
                 "\"col_8\" SMALLINT, " +
                 "\"col_9\" INTEGER, " +
                 "\"col_10\" REAL, " +
-                "\"col_11\" CHAR(128) " +
+                "\"col_11\" CHAR(12) " +
```

Comment:
looks unrelated?

Reply:
Actually it is. H2 always returns CHAR(n) with the trailing spaces.

Reply:
that is a correct behavior. every database should do that